test/common.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import (
+	"bytes"
+	"embed"
+	cmpact "encoding/json"
+	"github.com/goccy/go-json"
+	"fmt"
+	manifestsdk "github.com/TBD54566975/ssi-sdk/credential/manifest"
+	"github.com/oliveagle/jsonpath"
+	"github.com/pkg/errors"
+	credmodel "github.com/tbd54566975/ssi-service/internal/credential"
+	"io"
+	"net/http"
+)
+
+const (
+	endpoint = "http://localhost:8080/"
+	version  = "v1/"
+)
+
+var (
+	//go:embed testdata
+	testVectors embed.FS
+)
+
+func compactJSONOutput(json string) string {
+	jsonBytes := []byte(json)
+	buffer := new(bytes.Buffer)
+	if err := cmpact.Compact(buffer, jsonBytes); err != nil {
+		fmt.Println(err)
+	}
+
+	return buffer.String()
+}
+
+func getJSONElement(jsonString string, jsonPath string) (string, error) {
+	jsonMap := make(map[string]interface{})
+	if err := json.Unmarshal([]byte(jsonString), &jsonMap); err != nil {
+		return "", errors.Wrap(err, "problem with unmarshalling json string")
```

Comment:
Errors are typically already indicative of problems, so removing it would preserve the semantics.
```suggestion
		return "", errors.Wrap(err, "unmarshalling json string")
```